release notes/v0.47.0.md

```diff
@@ -0,0 +1,144 @@
+- [#3341](https://github.com/grafana/k6/pull/3341), [#3339](https://github.com/grafana/k6/pull/3339) Updates `goja`, includes runtime initialization speed-up and a fix for source indexes.
+- [#3311](https://github.com/grafana/k6/pull/3311) Updates the `alpine` image version that is used as the base of the k6 Docker image.
+- [browser#1043](https://github.com/grafana/xk6-browser/pull/1043), [browser#1021](https://github.com/grafana/xk6-browser/pull/1021), [browser#1019](https://github.com/grafana/xk6-browser/pull/1019), [browser#1014](https://github.com/grafana/xk6-browser/pull/1014) Fixes xk6-browser tests.
+- [browser#1000](https://github.com/grafana/xk6-browser/pull/1000), [browser#1024](https://github.com/grafana/xk6-browser/pull/1024) Refines xk6-browser issue and PR templates.
+- [browser#1003](https://github.com/grafana/xk6-browser/pull/1003), [browser#1009](https://github.com/grafana/xk6-browser/pull/1009), [browser#1010](https://github.com/grafana/xk6-browser/pull/1010) Internal changes to xk6-browser.
+- [browser#997](https://github.com/grafana/xk6-browser/pull/997) Updates xk6-browser readme.
+- [browser#962](https://github.com/grafana/xk6-browser/pull/962) CI fixes.
+
+## Roadmap
+
+### Native ECMAScript Modules support
+
+Work on this [epic issue](https://github.com/grafana/k6/issues/3265) has been picked up and there is some progress in the underlying implementation.
+
+One of the main internal changes will be dropping [Babel](https://babeljs.io), which is currently used to transpile [ESM](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) code to [CommonJS](https://en.wikipedia.org/wiki/CommonJS).
+
+For users, it will mean better JavaScript support as this change will automatically get [object spread](https://github.com/grafana/k6/issues/824) working and likely faster startup for *big* scripts. In the future, this also means JavaScript compatibility will be easier to add, since it'll only need to be supported in the JavaScript VM we use - [goja](https://github.com/dop251/goja).
+
+There's a risk that some k6 tests using both CommonJS and ECMAScript modules syntax will change in behavior. In practice, using both should never really be done as they're not compatible.
+
+Because there are risks involved, we are going to do this very carefully and only once we have done a lot of tests. That means this is currently planned for v0.49.0, but with the majority of the remaining work done in the v0.48.0 cycle.
+
+This way, we will also be able to have most of our users test these changes from our `master` branch, using Docker images, for example.
+
+### Future breaking changes
+
+It's relevant to stretch that in the next release several breaking changes will be applied as final stage of their deprecation period:
+
+- [#3065](https://github.com/grafana/k6/issues/3065) Limit the length of metric names to 128 characters and the set of the allowed symbols.
```

Comment:
Technically 128 characters is a limti now as well :thinking:

Reply:
Maybe we can be more verbose here and explain what happened? Something like:

"Metric names will be limited to 128 characters and a set of allowed symbols. This was going to be changed to a 63 character limit to ensure compatibility with Otel standards, but since then, the Otel limit has been updated to 255 characters. You can find more details about the changes in the linked issue."

Reply:
I think that is too much especially as we have had 1 report at this being a problem over the months taht we are doing thsi change. And this is already explained in the docs. 

So let's just keep it like this.